content/applications/hr/time_off/allocations.rst

```diff
@@ -64,7 +87,7 @@ information:
   Multiple selections can be made for either :guilabel:`Employees` or :guilabel:`Employee Tag`.
 
   Only one selection can be made for the :guilabel:`Company` or :guilabel:`Department`.
-- :guilabel:`Add a reason...`: if any description or note is necessary to explain the time off
+- :guilabel:`Add a reason...`: If any description or note is necessary to explain the time off
```

Comment:
In the Mode field, it explains the possible next field that appears (in this example, the employee field). So this is covered in the 'Mode' explanation since picking a mode determines what you see. It just doesn't have a separate bullet point since it was already explained. I added text that says "the field that appears beneath Mode - since it just said 'following field' originally.